pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -90,6 +90,8 @@ func runE(logger log.Logger, streams cmd.IOStreams, flags *flagpole) error {
 		cluster.CreateWithRetain(flags.Retain),
 		cluster.CreateWithWaitForReady(flags.Wait),
 		cluster.CreateWithKubeconfigPath(flags.Kubeconfig),
+		cluster.CreateWithDisplayUsage(true),
+		cluster.CreateWithDisplaySalutation(true),
```

Comment:
Allowing library users to avoid these separately from avoiding all output. (actually by default at the moment, not sure what the default should be ...)